model/src/main/resources/database.properties

```diff
@@ -1,4 +1,4 @@
 url=jdbc:postgresql://localhost:5432/testsys
 driver=org.postgresql.Driver
-user=vlad
-password=vlad
\ No newline at end of file
+user=sagolbah
+password=
```

Comment:
Perhaps, we can all set properties to:
```
url=jdbc:postgresql://localhost:5432/testsys
driver=org.postgresql.Driver
user=testsys_admin
password=
```
@vladrus13 @Sagolbah

Reply:
OK